pvlib/iotools/acis.py

```diff
@@ -0,0 +1,105 @@
+import requests
+import pandas as pd
+import numpy as np
+
+
+def get_acis_precipitation(latitude, longitude, start, end, dataset,
+                           url="https://data.rcc-acis.org/GridData", **kwargs):
+    """
+    Retrieve daily gridded precipitation data from the Applied Climate
```

Comment:
By saying "gridded" I'm getting the impression that I can retrieve multiple locations at the same time (gridded data). Perhaps replace gridded with modeled (if it is modeled data)?